virttest/unittest_utils/mock.py

```diff
@@ -1,15 +1,16 @@
-__author__ = "raphtee@google.com (Travis Miller)"
-
+from __future__ import print_function
 import collections
 import re
 import sys
-from __future__ import print_function
 
 import six
 from six import StringIO
 from six import string_types as basestring
 
 
+__author__ = "raphtee@google.com (Travis Miller)"
+
+
```

Comment:
yes, move `from __future__ import print_function` to the first line is must, and could you explain why move `__author__` ?

Reply:
@yanan-fu I don't want to intersperse other commands in the import site